core/util.cjs

```diff
@@ -77,31 +77,23 @@ class Util {
   }
 
   /**
-   * Returns a new LHR that's reshaped for slightly better ergonomics within the report rendereer.
-   * Also, sets up the localized UI strings used within renderer and makes changes to old LHRs to be
-   * compatible with current renderer.
-   * The LHR passed in is not mutated.
-   * TODO(team): we all agree the LHR shape change is technical debt we should fix
-   * @param {LH.Result} result
-   * @return {LH.ReportResult}
+   * Upgrades an lhr object in-place to account for changes in the data structure over major versions.
```

Comment:
Maybe add a note here mentioning that this doesn't verify lhr shape

Reply:
Wait, what about the name/interface makes that not clear already?

Reply:
Fair, it's probably fine.

Reply:
Ok so walk me through what was so wrong about trying to get information on that person who's hacking my account I figured I needed to know who he is he claims to be at my home all the time so wanted to see if he was local or out of state